templates/hello-world/contribution.ts

```diff
@@ -27,7 +27,7 @@ export class <%= params.extensionPrefix %>MenuContribution implements MenuContri
     registerMenus(menus: MenuModelRegistry): void {
         menus.registerMenuAction(CommonMenus.EDIT_FIND, {
             commandId: <%= params.extensionPrefix %>Command.id,
-            label: 'Say Hello'
+            label: <%= params.extensionPrefix %>Command.label
```

Comment:
I understand why the `label` was updated, however what do you think is best:
- we omit the `label` (it gets contributed directly from the `Command`) - less explicit example
- we include the `label` as it is more explicit.

I'll let you decide, I've tested the changes and they work correctly for me :+1: 
To test I did the following:
1. clone the repo, checkout the pull-request
2. build the example (`yarn`)
3. performed `npm link`
4. created a directory and used the generator to generate an extension
5. verified the changes

Reply:
I thought about this and concluded with the contributed solution, as it shows that you can override the label at this place if you like to...

Reply:
> I thought about this and concluded with the contributed solution, as it shows that you can override the label at this place if you like to...

I think that's a good idea, it shows extenders that they can provide a custom label (different than the command label) if they so choose.